samples/grid-pro/demo/todo-app/demo.html

```diff
@@ -0,0 +1,28 @@
+<script src="https://code.highcharts.com/dashboards/datagrid.js"></script>
+
+<div class="content" lang="pl">
```

Comment:
You probably forgot to delete `lang` here after debugging. 😄 
```suggestion
<div class="content">
```